docs/advanced/peering/advanced-offloading.md

```diff
@@ -0,0 +1,251 @@
+# Offloading in depth
+
+## Overview
+
+This document will go over the process of acquiring resources and making them available as a `Node` in the consumer cluster.
+
+You can add a `VirtualNode` to a consumer cluster in two different ways:
+
+1. By creating a `ResourceSlice` in the tenant namespace in the consumer cluster.
+2. By creating a `VirtualNode` in the consumer cluster.
+
+Note that the `ResourceSlice` method is the **preferred way** to add a `VirtualNode` to a consumer cluster, but it requires the *Authentication module* to be enabled (that is enabled by default).
```

Comment:
I would add in which case the authentication module is enabled by default (e.g. when peering is creating with the `liqoctl peer` command)